cgo.go

```diff
@@ -0,0 +1,11 @@
+// +build !customenv
+
+package gocv
+
+/*
+#cgo !windows pkg-config: opencv
+#cgo darwin   CXXFLAGS:   --std=c++1z -stdlib=libc++
```

Comment:
I don't think OpenCV requires C++17?
Why enforce the stlib used on macOS?

Reply:
I copied the values from `env.sh`. Just tested and it builds with just `--std=c++11`.

Reply:
That might have been an artifact I left in which trying to work on macOS builds at some point. Seems familiar. IN any case, best to only enforce the minimum needed version, right?

Reply:
It probably doesn't matter in practice, but I generally think the less flags the better. :)